src/System.Net.Http/tests/FunctionalTests/CancellationTest.cs

```diff
@@ -28,7 +28,7 @@ public CancellationTest(ITestOutputHelper output)
         [InlineData(false, true)]
         [InlineData(true, false)]
         [InlineData(true, true)]
-        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "dotnet/corefx #19038")]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.Uap | TargetFrameworkMonikers.UapAot | TargetFrameworkMonikers.NetFramework, "uap: dotnet/corefx #20010, netfx: dotnet/corefx #19038")]
```

Comment:
Yes, I will do that for the desktop tests that were disabled (temporary) to get a clean run. They all have an open issue with a disabled-test label. So I will scan those and update them to be ActiveIssue